frontend/app/components/Spots/SpotPlayer/components/AccessModal.tsx

```diff
@@ -35,70 +36,64 @@ function AccessModal() {
     spotStore.pubKey ? `?pub_key=${spotStore.pubKey.value}` : ''
   }`;
 
+  useEffect(() => {
+    if (spotStore.pubKey) {
+      const interval = Object.keys(expirationValues).find(key => expirationValues[key as Intervals] === spotStore.pubKey.expiration);
```

Comment:
this will never happen because expiration depletes

Reply:
updated